client/src/App.tsx

```diff
@@ -23,6 +23,15 @@ function App(): JSX.Element {
                 <Route exact path="/dashboard">
                   <Dashboard />
                 </Route>
+                <Route exact path="/myjobs">
```

Comment:
```suggestion
                <Route exact path="/my-jobs">
```
Please use hyphenated case for route names in JS